filters/data-test.properties

```diff
@@ -48,8 +48,10 @@ environment.build.hibernate.connection.validationQuery=select 1 from INFORMATION
 
 # uPortal server configuration properties
 environment.build.uportal.server=localhost:8080
+environment.build.real.uportal.server=localhost:8080
 environment.build.uportal.protocol=http
 environment.build.uportal.context=/uPortal
+environment.build.real.uportal.context=/uPortal
 environment.build.uportal.secureSessionCookie=false
```

Comment:
@bjagg : yes setting "real" to the original value from property would be better I will take in note to change it.
And for the name in that case I would prefer to rename it as "cluster" or "instance" or "local", in my mind the "casproxy" isn't appropriated as i'm using the host also for other properties. 
@drewwills : I can't do on an other way, I need to specify all cluster host and it's the only place at now, expect if all cas configuration go on a spring bean, I tried to do in that way but had problems with uPortal auth services implementation. 
The best would be to make a big clean on uportal auth schema/chain and use maybe something like spring-security, like that users will be able to adapt it on a way that would be a "standard" and generaly it's reusable/already used on an other app.